cmd/ebrelayer/events/event.go

```diff
@@ -17,23 +17,34 @@ import (
 
 // LockEvent : struct which represents a single smart contract event
 type LockEvent struct {
-	Id    [32]byte
-	From  common.Address
-	To    []byte
-	Token common.Address
-	Value *big.Int
-	Nonce *big.Int
+	Chain    string
+	Contract common.Address
```

Comment:
Can we name this BridgeContractAddress?

Reply:
https://github.com/swishlabsco/peggy/pull/4/commits/be6ce9d49dac7195bbcf2bf3b5260ebd2afc5715